debian/nnstreamer-python2.install

```diff
@@ -0,0 +1,2 @@
+/usr/lib/nnstreamer/filters/libnnstreamer_filter_python2.so
+/usr/lib/nnstreamer/filters/nnstreamer_python2.so /usr/lib/python2.7/dist-packages/
```

Comment:
The directory name, python2.7, is too specific. It is better to apply more general approach, IMO.

Reply:
Python 2 is equivalent to Python 2.7. So, we support python 2.7 & 3.x.